examples/examples/core_client.rs

```diff
@@ -42,7 +43,7 @@ async fn main() -> anyhow::Result<()> {
 
 	let (tx, rx) = WsTransportClientBuilder::default().build(uri).await?;
 	let client: Client = ClientBuilder::default().build_with_tokio(tx, rx);
-	let response: String = client.request("say_hello", None).await?;
+	let response: String = client.request("say_hello", rpc_params![]).await?;
```

Comment:
maybe we could have a type definition for empty params as well.... I reckon that some folks may not want to use the macro :P